contracts/KeeperRegistry.sol

```diff
@@ -356,8 +356,8 @@ contract KeeperRegistry is
   )
     external
     override
-    validUpkeep(id)
   {
+    require(s_upkeep[id].maxValidBlocknumber > block.number.add(CANCELATION_DELAY), "upkeep must be active");
```

Comment:
Maybe easier to just check that it hasn't been cancelled at all? We have a similar check in #cancelUpkeep.
```suggestion
    require(s_upkeep[id].maxValidBlocknumber != UINT64_MAX, "upkeep must be active");
```